### PR TITLE
[REF] Clean up return variables on updateContributionStatus, updatePendingOnlineContribution

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Minor preliminary cleanup

Before
----------------------------------------
Values returned & not used

After
----------------------------------------
Values not returned

Technical Details
----------------------------------------
The return variable from updatePendingOnlineContribution is never used - ergo don't return it. This means
the return value from updateContributionStatus is also never used - ergo - you get it. In addition don't call the function
with empty variables - I've moved the check as to whether they are empty into the calling function - although likely
in a second pass the checks will be removed or change again

Comments
----------------------------------------

Ideally we want to stop using BaseIPN from this class. I took a look at the $baseIPN->cancelled code & I don't think it does anything that an api call wouldn't in this case so I'm inclined to simply do that - that would be the next follow up

### CODE REVIEW

File: CRM/Event/Form/Task/Batch.php [hunk 1]
```diff
@@ -260,14 +260,10 @@ public function postProcess() {
    * @param int $participantId
    * @param int $statusId
    *
-   * @return mixed
    * @throws \CRM_Core_Exception
    * @throws \CiviCRM_API3_Exception
    */
   public static function updatePendingOnlineContribution($participantId, $statusId) {
-    if (!$participantId || !$statusId) {
```
Comment (eileenmcnaughton) [inline CRM/Event/Form/Task/Batch.php:268]: moved to the calling function - don't call a function if you don't have the variables

File: CRM/Event/Form/Task/Batch.php [hunk 2]
```diff
@@ -513,7 +500,7 @@ public function submit($params) {
         if ($statusChange) {
           CRM_Event_BAO_Participant::transitionParticipants([$key], $value['status_id'], $fromStatusId);
         }
-        if ($relatedStatusChange) {
+        if ($relatedStatusChange && $key && $value['status_id']) {
```
Comment (eileenmcnaughton) [inline CRM/Event/Form/Task/Batch.php:503]: don't call the function if you don't have the variables

File: CRM/Event/Form/Task/Batch.php [hunk 3]
```diff
@@ -332,10 +325,6 @@ public static function updateContributionStatus($params) {
     $componentName = $params['componentName'] ?? NULL;
     $contributionId = $params['contribution_id'] ?? NULL;
 
-    if (!$contributionId || !$componentId || !$componentName || !$statusId) {
```
Comment (eileenmcnaughton) [inline CRM/Event/Form/Task/Batch.php:335]: See last line

**Comment by mattwire** [comment]
Makes sense

**Pull request merged**